httpclient/httpclient.go

```diff
@@ -164,14 +164,15 @@ type Request struct {
 	body    interface{} // If set this will be sent as JSON
 	rawBody []byte      // If set this will be sent as is
 
-	decoders map[int]decoder          // If set will be used to decode the response body by http status code
-	headerFn func(header http.Header) // If set will be called with the response header
-	cookie   *http.Cookie
-	headers  map[string]string
-	timeout  time.Duration // The individual per call timeout
-	retry    bool
-	query    url.Values
-	rawquery string
+	decoders       map[int]decoder          // If set will be used to decode the response body by http status code
+	headerFn       func(header http.Header) // If set will be called with the response header
+	cookie         *http.Cookie
+	headers        map[string]string
+	timeout        time.Duration // The individual per call timeout
+	maxElapsedTime time.Duration // The total timeout for the entire request including retries
```

Comment:
~ `maxDuration` mebe - `Time` on the end kinda makes it sound like an absolute time

Reply:
nah, i see this naming matches the `backoff` naming ignore me